tethysapp/gwdm/public/js/update_variable.js

```diff
@@ -60,11 +60,11 @@ var LIBRARY_OBJECT = (function() {
 
 
     var deleteIcon = function(cell, formatterParams){ //plain text value
-        return "<span class='glyphicon glyphicon-remove'></span>";
+        return "<span class='x-lg'></span>";
```

Comment:
Need to change `<span>` to `<i>` and add `bi` to classes:

e.g.:
```
return "<i class='bi bi-x-lg'></i>";
```

Fix all instances of this in this file.